week-2/E-conditionals/exercise.js

```diff
@@ -10,10 +10,14 @@ var role = "mentor";
 
 var name2 = "Harun";
 var role2 = "student";
+if (name2){
+  return"hi , name ,is "+name2;
+}
 
 
 console.log("Hi, My name is " + name);
 console.log("Hi, My name is " + name2);
+console.log("hi, we are student "+ role2);
```

Comment:
Does this only show the student name if you run it? Also `if(name2)` won't do anything.